### DISCUSSION TITLE
Lazily registering updaters in graphcache?

### DISCUSSION THREAD

**Discussion by tjk** [urql-graphql/urql#1868] [Q&A] [answered]
It looks like the `cacheExchange` store needs to be constructed with all the `updates` upfront. Would it make sense to be able to register some lazily? (It would help with code splitting unless there is a better strategy I'm unaware of..)

**Comment by JoviDeCroock** [comment, marked as answer]
If you really want to you could do this similar to how it was handles in Redux where you mutate an object

```js
export const updates = { current: {} }
cacheExchange({ updates: updates.current })
```

Then when you load in a new module you can mutate the object, however this feels quite error-prone for the minimal benefit you'll have in terms of b-size.
